awslambda/cml/labbing_card.json

```diff
@@ -0,0 +1,71 @@
+[{"contentType":
+  "application/vnd.microsoft.card.adaptive",
+  "content":
+  {
+    "type": "AdaptiveCard",
+    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+    "version": "1.2",
+    "body": [
+        {
+            "type": "RichTextBlock",
+            "spacing": "None",
+            "inlines": [
+                {
+                    "type": "TextRun",
+                    "text": "Are you still labbing? (Select labs to keep)"
+                }
+            ]
+        },
+        {
+            "type": "Input.ChoiceSet",
+            "id": "labIds",
+            "style": "expanded",
+            "spacing": "Small",
+            "isMultiSelect": true,
+            "choices": {{lab_choices}}
```

Comment:
small nit - typically you want to write this as `{{ lab choices }}` to make it a little more readable.  Likewise with the rest of them